cms/djangoapps/contentstore/views/tests/test_transcripts.py

```diff
@@ -1,17 +1,18 @@
 """Tests for items views."""
 
 import copy
+from codecs import BOM_UTF8
```

Comment:
why not `BOM`? why `BOM_UTF8`?

Reply:
@muhammad-ammar BOM is an alias for BOM_UTF16: https://docs.python.org/2/library/codecs.html